librz/socket/run.c

```diff
@@ -655,52 +656,130 @@ RZ_API bool rz_run_parseline(RzRunProfile *p, const char *b) {
 	return true;
 }
 
-RZ_API const char *rz_run_help(void) {
```

Comment:
This function doesn't really belong here, could you please move it to the `librz/main/rz-run.c`?

Reply:
Sure.
I just make a static function in `librz/main/rz-run.c` similar to how `static int rasm_show_help(int v)` in `rz-asm`?

Reply:
Yep.